bin/desi_fvc_proc

```diff
@@ -247,6 +252,27 @@ for seqid,spots in enumerate(spots_list) :
                     spots[k] = np.zeros(len(spots))
             spots[k][selection]=expected_pos[k][indices_of_expected_pos]
 
+    if args.expected_positions is not None and ( args.turbulence_correction or args.turbulence_correction_with_pol ):
+
+        if args.turbulence_correction_with_pol :
+            log.info("Turbulence correction (local polynomial fit) ...")
+        else :
+            log.info("Turbulence correction (gaussian processes) ...")
+        selection=(spots["LOCATION"]>=0)&(spots["X_FP_EXP"]!=0)&(spots["Y_FP_EXP"]!=0) # matched
+
+        if args.turbulence_correction_with_pol :
+            new_x,new_y = correct_with_pol(spots["X_FP"][selection],spots["Y_FP"][selection],spots["X_FP_EXP"][selection],spots["Y_FP_EXP"][selection])
+        else :
+            new_x,new_y = correct(spots["X_FP"][selection],spots["Y_FP"][selection],spots["X_FP_EXP"][selection],spots["Y_FP_EXP"][selection])
+        rms_before = np.sqrt(np.mean((spots["X_FP"][selection]-spots["X_FP_EXP"][selection])**2+(spots["Y_FP"][selection]-spots["Y_FP_EXP"][selection])**2))
+        rms_after  = np.sqrt(np.mean((new_x-spots["X_FP_EXP"][selection])**2+(new_y-spots["Y_FP_EXP"][selection])**2))
+        log.info("rms(measured-expected)={:5.4f}mm rms(corrected-expected)={:5.4f}mm ".format(rms_before,rms_after))
+        if rms_before < rms_after :
+            log.warning("turbulence correction does not reduce the rms?")
+            # apply it anyway because there might be a good reason for this
```

Comment:
This is a good check.  For what it's worth, I don't expect this to happen.  We're basically fitting and removing correlated signals... I guess in the limit that there is no turbulent contribution we could just add a small amount of noise.